android/quest/build.gradle.kts

```diff
@@ -321,11 +321,18 @@ android {
       manifestPlaceholders["appLabel"] = "PSI WFA"
     }
 
-    create("eusm") {
+    create("eusm_mg") {
```

Comment:
@ndegwamartin @ellykits DId we ever agree on how to name these?  Are we using underscores or camel case

Reply:
Use camel case

Reply:
@marklosh  Here is the documentation https://docs.opensrp.io/engineering/app/automated-releases#flavors

Reply:
noted @Mwalimu

we can update the naming, we'll be good to go